virtualizarr/readers/hdf/hdf.py

```diff
@@ -371,6 +371,10 @@ def _virtual_vars_from_hdf(
             group_name = "/"
 
         variables = {}
+        non_coordinate_dimesion_vars = HDFVirtualBackend._find_non_coord_dimension_vars(
+            group=g
+        )
+        drop_variables = list(set(drop_variables + non_coordinate_dimesion_vars))
```

Comment:
nitpick but I think this is more legible and faster:
```suggestion
        drop_variables = [*drop_variables, *non_coordinate_dimension_vars]
```

Reply:
Speed of this is not important, and I think the first one makes intent clearer (if the intent is to de-duplicate using set?)

Reply:
Good point! I was getting too nitpicky. Just the typo then :)